python/pyarrow/array.pxi

```diff
@@ -15,6 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
+import pyarrow.pandas_compat as pdcompat
```

Comment:
Pandas is an optional dependency of pyarrow, that's why pandas is not imported.

Reply:
I changed it to an import iff pandas is available